### ISSUE TITLE
Can't transfer watched items to new server

### ISSUE THREAD

**Issue by KillahB33** [linuxserver/docker-plex#241]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

Hi All, I had to migrate my plex docker to a new vm.
My nfs mounts are all exactly the same. I have copied over the config from the old machine. But I still can't get my watch history restored. I tried doing a database restore with moving com.plexapp.plugins.library.db and the blob one and that's not bringing my history back either.
------------------------------

## Expected Behavior
I would like all my watch history to be restored.

## Current Behavior
No watch history

## Steps to Reproduce

1.Migrate configs to a new machine (I even have the same ip as before)

## Environment
**Old OS:** CoreOS
New OS: CentOS
**CPU architecture:** x86_64
**How docker service was installed:**

## Command used to create docker container (compose)
  plex:
    image: linuxserver/plex
    container_name: plex
    volumes:
     - /opt/plex/config:/config
     - /opt/Movies:/data/movies
     - /opt/4kMovies:/data/4kmovies
     - /opt/Shows:/data/tvshows
     - /opt/plex/transcode:/data/transcode
    restart: always
    environment:
     - PUID=1000
     - PGID=1000
     - VERSION=latest
     - TZ=America/Toronto
    network_mode: host

## Docker logs
`usermod: no changes
          _         ()
         | |  ___   _    __
         | | / __| | |  /  \ 
         | | \__ \ | | | () |
         |_| |___/ |_|  \__/

Brought to you by linuxserver.io
To support LSIO projects visit:
https://www.linuxserver.io/donate/
GID/UID

User uid:    1000
User gid:    1000

[cont-init.d] 10-adduser: exited 0.
[cont-init.d] 40-chown-files: executing... 
[cont-init.d] 40-chown-files: exited 0.
[cont-init.d] 45-plex-claim: executing... 
[cont-init.d] 45-plex-claim: exited 0.
[cont-init.d] 50-gid-video: executing... 
[cont-init.d] 50-gid-video: exited 0.
[cont-init.d] 60-plex-update: executing... 
No update required
[cont-init.d] 60-plex-update: exited 0.
[cont-init.d] 99-custom-scripts: executing... 
[custom-init] no custom files found exiting...
[cont-init.d] 99-custom-scripts: exited 0.
[cont-init.d] done.
[services.d] starting services
Starting Plex Media Server.
[services.d] done.

`

**Comment by james-d-elliott** [comment]
If you copy the Plex config folder from one server to another it will transfer your watch history. You need to obviously make sure it's from the correct location to the correct location, and that the destination files are chowned properly. 

If you've done this the likely problem is the location of the files, I believe with LSIO's plex the location is as follows:
`/config/Library/Application\ Support/Plex\ Media\ Server/`

Which should have no files under Library or Application Support, except for the single subfolder. And under Plex Media Server it should have the folders: Cache, Codecs, Logs, Media, Metadata, and the file Preferences.xml (there may be more files/folders, but none of them should be named Library, Application Support, or Plex Media Server).

**Comment by KillahB33** [comment]
Yep that's exactly what I did, copied the config folder from one machine to the other. I have everything mounted correctly. When I first logged on it maid me claim the server, so I did that and then I could see my old server with my new one, so that was the first sign that things weren't working. I then setup the libraries again, hoping that with both of them there that it would work, it didn't. Then I copied over the config again and I was in the exact same place.
The only thing I can think of is doing a docker rm to delete the container but I don't want to put more time into this than I need to.
